distributed.nanny - INFO -         Start Nanny at: 'tcp://10.120.40.89:44465'
distributed.worker - INFO -       Start worker at:   tcp://10.120.40.89:44073
distributed.worker - INFO -          Listening to:   tcp://10.120.40.89:44073
distributed.worker - INFO -          dashboard at:         10.120.40.89:42528
distributed.worker - INFO - Waiting to connect to:   tcp://10.120.43.21:33949
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    8.00 GB
distributed.worker - INFO -       Local Directory: /home/ad/alberta/dask-worker-space/worker-bisuz3oj
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://10.120.43.21:33949
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
=>> PBS: job killed: walltime 14420 exceeded limit 14400
distributed.dask_worker - INFO - Exiting on signal 15
distributed.nanny - INFO - Closing Nanny at 'tcp://10.120.40.89:44465'
distributed.nanny - INFO - Worker process 15474 was killed by signal 15
Traceback (most recent call last):
  File "/home/ad/alberta/.conda/envs/perf-pangeo/lib/python3.6/multiprocessing/queues.py", line 240, in _feed
    send_bytes(obj)
  File "/home/ad/alberta/.conda/envs/perf-pangeo/lib/python3.6/multiprocessing/connection.py", line 200, in send_bytes
    self._send_bytes(m[offset:offset + size])
  File "/home/ad/alberta/.conda/envs/perf-pangeo/lib/python3.6/multiprocessing/connection.py", line 404, in _send_bytes
    self._send(header + buf)
  File "/home/ad/alberta/.conda/envs/perf-pangeo/lib/python3.6/multiprocessing/connection.py", line 368, in _send
    n = write(self._handle, buf)
BrokenPipeError: [Errno 32] Broken pipe
distributed.dask_worker - INFO - End worker
